# 01 - Federated Patient Stratification with AE - Biostatistik und KI in der Medizin

## Group Members

Pia Kober, Mohammadreza Ghassemzadeh, David Egelhofer, Bernard Burdiek

## Resources

### Datasets
https://drive.google.com/drive/folders/1DH4axgfDkEonjYVRfhovOShjPIJ6ZilR?usp=sharing 
The folder includes three simulated datasets (more can be generated if needed). Also, you can test the central approach on real datasets (in the same folder):
JHU cohort  https://pdc.cancer.gov/pdc/study/PDC000110 
PNNL cohort  https://pdc.cancer.gov/pdc/study/PDC000118 
### References
You can try models that were developed for scRNAseq data: 
A comparison of deep learning-based pre-processing and clustering approaches for single-cell RNA sequencing data. Briefings in Bioinformatics, Volume 23, Issue 1, January 2022, bbab345, https://doi.org/10.1093/bib/bbab345 
And models and evaluation methods from this benchmark:
Leng, D., Zheng, L., Wen, Y. et al. A benchmark study of deep learning-based multi-omics data fusion methods for cancer. Genome Biol 23, 171 (2022). https://doi.org/10.1186/s13059-022-02739-2 

Federated AE and VAE â€” may not be suitable for this specific task, but may give an impression of how to implement it in a federated manner in general:
M. Polato, "Federated Variational Autoencoder for Collaborative Filtering," 2021 International Joint Conference on Neural Networks (IJCNN), Shenzhen, China, 2021, pp. 1-8, doi: 10.1109/IJCNN52387.2021.9533358. 
An autoencoder-based confederated clustering leveraging a robust model fusion strategy for federated unsupervised learning https://doi.org/10.1016/j.inffus.2024.102751
FREPD: A Robust Federated Learning Framework on Variational Autoencoder https://www.techscience.com/csse/v39n3/44055 
